js/App.vue

```diff
@@ -53,47 +53,44 @@
-                                    <span class="dotted-underline text-yellow-900 cursor-pointer" @click.prevent="formFill('chesscom', 'Hikaru')"
-                                        >Hikaru Nakamura</span
-                                    >,
-                                    <span
-                                        class="dotted-underline text-yellow-900 cursor-pointer"
-                                        @click.prevent="formFill('chesscom', 'MagnusCarlsen')"
-                                        >Magnus Carlsen</span
-                                    >,
-                                    <span
-                                        class="dotted-underline text-yellow-900 cursor-pointer"
-                                        @click.prevent="formFill('chesscom', 'VampireChicken')"
-                                        >Jonathan Schrantz</span
-                                    >,
-                                    <span class="dotted-underline text-yellow-900 cursor-pointer" @click.prevent="formFill('chesscom', 'IMRosen')"
-                                        >Eric Rosen</span
-                                    >,
-                                    <span
-                                        class="dotted-underline text-yellow-900 cursor-pointer"
-                                        @click.prevent="formFill('chesscom', 'DanielNaroditsky')"
-                                        >Daniel Naroditsky</span
-                                    >
-                                    or
-                                    <span class="dotted-underline text-yellow-900 cursor-pointer" @click.prevent="formFill('chesscom', 'GothamChess')"
-                                        >GothamChess</span
-                                    >
-                                    <br />
-                                    On Lichess:
-                                    <span class="dotted-underline text-yellow-900 cursor-pointer" @click.prevent="formFill('lichess', 'zolpi')"
-                                        >Jonathan Schrantz</span
-                                    >,
-                                    <span class="dotted-underline text-yellow-900 cursor-pointer" @click.prevent="formFill('lichess', 'EricRosen')"
-                                        >Eric Rosen</span
-                                    >
-                                    or
-                                    <span
-                                        class="dotted-underline text-yellow-900 cursor-pointer"
-                                        @click.prevent="formFill('lichess', 'DrNykterstein')"
-                                        >Magnus Carlsen</span
-                                    >
+                                <div class="text-sm">Examples
+                                    <ul class="list-none list-inside pl-4">
+                                        <li>Chess.com:
```

Comment:
There is actually a good reason I don't like to have lists here,  in terms of design and accessibility for the users, it's better that the initial form does not appear very long (in terms of height). Users should not need to scroll down in order to click "analyze my games". 

I tried to do a more suitable way in my version, even tho I still prefer the initial version: It's only one line, it's not mandatory, it does not add more space in the first form.

Reply:
You make a good point. I just finished learning JavaScript in university a month ago, so I'm unfamiliar with best practices. Technically these contributions are my first go at it!

Reply:
Thanks for the contributions! Trying and erring is the most effective way of learning.